### DISCUSSION TITLE
发现TTS server内置的三个接口，仅微软大声朗读可以使用，其他的两个接口无法使用，还有导入导出配置导出，配置没问题，导入配置有bug，导入不进去，还有编辑条目或者添加条目的时候，那个有一个试听按钮没有无障碍未加标签，希望增加无障碍标签

### DISCUSSION THREAD

**Discussion by zmyznu** [jing332/tts-server-android#24] [General]
希望修复一下

**Comment by jing332** [comment]
azure演示接口暂不可用

**Reply by zmyznu** [reply]
好的，谢谢开发者，我只希望那两个功能抓紧优化一下

**Comment by jing332** [comment]
如果你没使用azure接口，就得附上完整日志，不然谁知道具体原因。

**Comment by jing332** [comment]
emmm😅你都把内容写到标题上了，邮箱提醒一时间没看到

**Comment by jing332** [comment]
导入配置确实有问题，
azure只能等微软修复

**Reply by zmyznu** [reply]
希望下个版本可以修复导入问题，因为导致问题出错，我只能卸载了新版本，然后幸亏有之前的包给备份回来了，要不然就惨了 n

**Comment by jing332** [comment]
对了，APP还有哪里需要无障碍优化吗？

**Reply by zmyznu** [reply]
目前我感觉无障碍现在已经很好了，暂时不要优化了，然后现在主要是你这个开发版太难获取到了，有没有什么简便的方式获取到呢？因为头会用 github的你发的那个开发版下载地址不大会弄那个东西，谢谢

**Reply by jing332** [reply]
> 目前我感觉无障碍现在已经很好了，暂时不要优化了，然后现在主要是你这个开发版太难获取到了，有没有什么简便的方式获取到呢？因为头会用 github的你发的那个开发版下载地址不大会弄那个东西，谢谢
> 

打开Actions，点击顶部的最新提交（比如当前最新为"perf: TtsEdit UI"），然后下滑到底部，点TTS-Server字样的文件下载，最后解压zip安装即可。

**Comment by jing332** [comment]
开发版已经修复了，直接更新

**Comment by jing332** [comment]
这也是为了提高门槛，开发版易出bug，小白很容易摸不着头脑。

**Comment by zmyznu** [comment]
那您能否告诉我们怎么下载，因为这儿太难下载了

**Reply by limne** [reply]
登录自己的github账号就可以下载action开放版了，release版不需要登录。
